Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM        python:3.5
+MAINTAINER  Michael B. Klein <michael.klein@northwestern.edu>
+RUN         mkdir -p /var/app /data
+WORKDIR     /var/app
```

Comment:
I believe the UNIX standard is to use `/var` for data files and `usr` for application data. Unless you are familiar with different setups, shall this be set to `usr/local`?